docs/sources/datasources/loki.md

```diff
@@ -6,31 +6,25 @@ aliases = ["/docs/grafana/latest/features/datasources/loki"]
 weight = 800
 +++
 
-# Using Loki in Grafana
+# Loki data source
 
-> BETA: Querying Loki data requires Grafana's Explore section.
-> Grafana v6.x comes with Explore enabled by default.
-> In Grafana v5.3.x and v5.4.x. you need to enable Explore manually.
-> Viewing Loki data in dashboard panels is supported in Grafana v6.4+.
+Grafana ships with built-in support for Loki, an open source log aggregation system by Grafana Labs. Add it as a data source and you are ready to build dashboards or query your log data in [Explore]({{< relref "../explore" >}}).
 
-Grafana ships with built-in support for Loki, Grafana's log aggregation system.
-Just add it as a data source and you are ready to query your log data in [Explore]({{< relref "../explore" >}}).
+This topic explains options, variables, querying, and other options specific to this data source. Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
 
-## Adding the data source
+> **Note:** If you have issues with getting this data source to work and Grafana is giving you undescriptive errors, then check your log file (try looking in /var/log/grafana/grafana.log).
```

Comment:
```suggestion
> **Note:** To troubleshoot configuration and other issues, check the log file (/var/log/grafana/grafana.log).
```

Reply:
This path is not going to work for all users. On my Windows machine, it's in grafana/data/log/grafana.log. I don't have a var folder.